downloader/src/main/kotlin/vcs/Subversion.kt

```diff
@@ -206,16 +206,21 @@ class Subversion : VersionControlSystem(), CommandLineTool {
                     getWorkingTree(File(targetDir, pkg.vcsProcessed.path))
                 }
             } else {
+                // The path to the tag (including the tag name), relative to the repository root.
                 val tagPath: String
+
+                // The path within the tag to limit the working directory to, relative to the repository root.
                 val path: String
 
-                val tagsIndex = pkg.vcsProcessed.path.indexOf("tags/")
-                if (tagsIndex == 0 || (tagsIndex > 0 && pkg.vcsProcessed.path[tagsIndex - 1] == '/')) {
+                val tagsMarker = "tags/"
```

Comment:
`tagMarker`  could go to companion or file scope to get rid of one line here.?

Reply:
I plan to remove this whole function anyway soon; and if not, I'll fix that later.